docs/v0.5/en/user-guide/operations/configuration.md

```diff
@@ -451,6 +451,42 @@ append_stdout = true
 
 How to use distributed tracing, please reference [Tracing](./tracing.md)
 
+### Export metrics options
+
+`frontend`, `metasrv`, `datanode` and `standalone` support collecting metrics generated by themselves and sending the collected metrics to [Prometheus RemoteWrite protocol](https://prometheus.io/docs/concepts/remote_write_spec/) compatible receiver, such as sending to `greptimedb` itself. **Note: Export metrics has nothing to do with using Prometheus to scape metrics generated by `greptimedb`. Export metrics is only used as a way for `greptimedb` to export the metrics generated by itself.** Users can configure related export parameters through the `[export_metrics]` section.
```

Comment:
There are three things here:
1. > Export metrics has nothing to do with using Prometheus to scape metrics generated by `greptimedb`.

    what does it mean? can you revise the sentence more simpler and clearer?

2. Please use the following format to write NOTE
:::tip NOTE
xxxx...
:::

3. > Users can configure related export parameters through the `[export_metrics]` section.

    Users -> You